packages/icons/src/Icon/iconTypes.ts

```diff
@@ -41,4 +41,25 @@ export const iconTypes = {
     warningCircle: require('./assets/warningCircle.svg'),
 } as const;
 
+export const coloredIconTypes = {
+    BCH: require('./coloredAssets/BCH.svg'),
```

Comment:
please do not use capitals, in current suite code we don't do that for any crypto shortcuts

I am also thinking we should prefix all crypto icons with some prefix like `cryptoLogo` and flags with `flag`. 

Another options could be create something like two components `CryptoIcon` and `FlagIcon`, which actually makes sense in case there will be only these two types of colored assets, which looks like that. What do you think?

Reply:
I wanted to keep capitals because file names _btc.svg_, _dgb.svg_, etc. seemed to me like names typed with a broken keyboard :). But it is much better with the prefix `cryptoLogo`, good idea 👍 . The thing about creating two components seems good to distinguish what kind of component is occurring in a code. `CryptoIcon` says more than `ColoredIcon`... we probably don't need a component with a generic name for any of our use cases...

Reply:
So you agree with two component approach? If so we should probably change file structure a bit. I would suggest to place `assets` folder to package root (not inside of src folder) a then create like three folders in it (icons, cryptoIcons, flags). This is flat enough for me :D 

Then you will have like Icon, CryptoIcon, FlagIcon and I think maybe we will also add TrezorDevice icon in future.